docs/api-reference/base-layer.md

```diff
@@ -26,10 +26,11 @@ of deck.gl, a check will be in place to enforce this
 
 ## Constructor
 
-- `props` (Object) - Layer properties.
-
 ### Basic Properties
 
+##### `props` (Object)
```

Comment:
This looks strange, I think this was intended to be the parameter list of the constructor

I think it is better to move this back and add a heading `Parameters`:

## Constructor

Initializes a new `Layer` instance.

Parameters:
- `props` (Object) - Layer properties.